### PR TITLE
fix(webpack-cli): handle promise rejection with package installation

### LINKED ISSUE #1283: Handle the promise rejection if the user interrupts in between package installation prompt
**Describe the bug**
Say the user fires in a command, `webpack-cli init` which results in the respective prompt showing up:-

```sh
✖ Would you like to install @webpack-cli/init? (That will run yarn add -D @webpack-cli/init) (Y/n) ·
```
And if the user interrupts the regular workflow, a promise rejection occurs which is left unhandled. 

**What is the current behavior?**
```sh
[webpack-cli] Promise rejection:
```

**Expected behavior**
Handles the promise rejection thereby replacing what is currently logged with an appropriate message.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
bugfix

**Did you add tests for your changes?**
Nope

**If relevant, did you update the documentation?**
N/A

**Summary**
closes #1283 

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
@ematipico it requires to mock the keyboard interrupts, and is gonna consume some sort of third party libraries for the purpose.

**Comment by ematipico** [comment]
With jest you can mock libraries or modules. I believe it's doable

File: packages/package-utils/__tests__/index.test.ts [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+
+import { packageExists, promptInstallation } from '@webpack-cli/package-utils';
+import ExternalCommand from '../../webpack-cli/lib/commands/ExternalCommand';
 
 describe('@webpack-cli/package-utils', () => {
     it('should check existence of package', () => {
+    	(packageExists as any).mockImplementation(() => true);
         const exists = packageExists('@webpack-cli/info');
-
+
```
Comment (snitin315) [inline packages/package-utils/__tests__/index.test.ts]: Why?

Comment (jamesgeorge007) [reply packages/package-utils/__tests__/index.test.ts]: Oops!

**Pull request merged**